components/enrichers/reachability/internal/conf/conf.go

```diff
@@ -0,0 +1,59 @@
+package conf
```

Comment:
perhaps this should be part of the base enricher?

Reply:
What do you mean by that?